loadsave.c

```diff
@@ -51,7 +57,17 @@ create_directory_listing(uint8_t *data)
 	*data++ = 'C';
 	*data++ = 0;
 
-	if (!(dirp = opendir("."))) {
+	#if __APPLE__ && (TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE)
```

Comment:
You do this concatening a few times in this file.
Maybe extract it to some static helper function?
You can move the ifdef to it too.

Reply:
👍 thanks I'll take a look.